File: backup/py_AFD/Token.py
class Token:
    def __init__(self, tipo, lexema, linha, coluna):
        self.lexema = lexema
        self.tipo = tipo
        self.linha = linha
        self.coluna = coluna

    def get_lexema(self):
        return self.lexema

    def get_tipo(self):
        return self.tipo

    def __str__(self):
        return "<" + self.lexema + ", " + self.tipo + ">"

File: src/alvaro/Lexico/Token.java
package Lexico;

public class Token {
    private String lexema;
    private String tipo;
    private int linha, column;

    public Token(String tipo, String lexema, int linha, int column) {
        this.lexema = lexema;
        this.tipo = tipo;
        this.linha = linha;
        this.column = column;
    }

    public String getLexema() {
        return lexema;
    }

    public String getTipo() {
        return tipo;
    }

    public int getLinha() {
        return linha;
    }

    public int getColumn() {
        return column;
    }

    @Override
    public String toString() {
        return "<" + tipo + ", " + lexema + ", " + linha + ", " + column + ">";
    }
}
